ui/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "scripts": {
     "dev": "vite",
-    "build": "pnpm generate:routes && tsc && vite build",
+    "build": "pnpm generate:api && pnpm generate:routes && tsc && vite build",
```

Comment:
Does pnpm accept multiple scripts to run in one invocation, like `pnpm generate:api generate:routes`?

Reply:
[It does](https://pnpm.io/cli/run#running-multiple-scripts), but I'd say the current is clearer than regex, so I'd go with it.

Reply:
> clearer than regex

Huh? What regex?

Reply:
Ah, I though you were referring to my example, which does not use a regex, but lists both scripts explicitly. If that's not supported, but only via regex, it could indeed be clearer to keep the current code.

Reply:
Yeah, I don't think just lining them up works, as you can give further arguments to scripts. Though I haven't tried out, but it's not something I've bumped into in the wild.

Reply:
Although `pnpm run "/^generate:.*/"` does have some charm, as it nicely runs any current and future generate scripts.

BTW, why do the docs say "pnpm *run*", but we don't in line 9?

Reply:
https://github.com/pnpm/pnpm/issues/2169 but apparently not in the web docs.